qiskit_optimization/algorithms/admm_optimizer.py

```diff
@@ -290,7 +289,9 @@ def solve(self, problem: QuadraticProgram) -> ADMMOptimizationResult:
         problem = int2bin.convert(problem)
 
         # we deal with minimization in the optimizer, so turn the problem to minimization
-        problem, sense = self._turn_to_minimization(problem)
+        from ..converters.maximize_to_minimize import MaximizeToMinimize
```

Comment:
Is there any problem with having the import at the head of the file with the others. Sometimes we do have these as inline imports later, but its normally to avoid some cyclic import problem that would occur otherwise, if there was no other realistic way to deal with it.

Reply:
The inline imports were added similar to the existing ones a few lines before:
`from ..converters.integer_to_binary import IntegerToBinary` 
I will put the imports of MaximizeToMinimize always to the head of the files, as this works properly.

Reply:
Fixed imports